Update buffer size (#34923)
dotnet/AspNetCore.Docs: aspnetcore/host-and-deploy/iis/advanced.md

</aspNetCore>
```

:::moniker-end

:::moniker range="<= aspnetcore-9.0"

> Configure the managed stack size using the `stackSize` setting in bytes in the `web.config` file. The default size is 17,825,792 bytes (17 MB). The following example changes the stack size to 100,000 hex, (1 MB)):

Author: ```suggestion
Configure the managed stack size using the `stackSize` setting in bytes in the `web.config` file. The default size is 17,825,792 bytes (17 MB). The following example changes the stack size to 100,000 hex, (1 MB):
```

> Configure the managed stack size using the `stackSize` setting in bytes in the `web.config` file. The default size is 17,825,792 bytes (17 MB). The following example changes the stack size to 1,048,576 bytes (1 MB):

Author: ```suggestion
Configure the managed stack size using the `stackSize` setting in bytes in the `web.config` file. The default size is 17,825,792 bytes (17 MB). The following example changes the stack size to 100,000 hex, (1 MB)):
```

```xml
<aspNetCore processPath="dotnet"
    arguments=".\MyApp.dll"
    stdoutLogEnabled="false"
    stdoutLogFile="\\?\%home%\LogFiles\stdout"
    hostingModel="inprocess">
  <handlerSettings>

>     <handlerSetting name="stackSize" value="1048576" />

Author: ```suggestion
    <handlerSetting name="stackSize" value="100000" />
```
